### PR TITLE
Removed default values for eslintPath and prettierPath

### PR DESCRIPTION
**Pull request by fredrikekelund**
**What**:
Removed default values for eslintPath and prettierPath. This improves compatibility for scenarios where not all dependencies of a project reside in a completely flat node_modules folder (the most common cause for this would be that npm 2.x is still being used)

**Why**:
These values are not required by prettier-eslint-cli itself. They are only passed to prettier-eslint, and even then, they're optional. When using npm >3, whether we pass the value or not is all the same - the node_modules directory is flattened and both prettier and eslint will reside in the same node_modules folder no matter which other module depends on it. But for npm 2.x, the default assumptions don't work.

It's true that people should be running npm >3 today, but for CI scenarios, npm 2 or downwards can still be used in combination with node 4.x (which is going to be supported for a long time still).

In any case, not passing a default value seems to align better with node's `require` algorithm - it will start where it needs to start (in this case wherever the prettier-eslint dependency has been installed), and then mimick find-up's logic.

**How**:
I simply removed the default values and the associated helper function.

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -160,23 +158,6 @@ const parser = yargs
 
 export default parser
 
-function getPathInHostNodeModules(module) {
-  logger.debug(`Looking for a local installation of the module "${module}"`)
-  const modulePath = findUp.sync(`node_modules/${module}`)
```
Comment (kentcdodds) [inline src/parser.js]: Looks like you can remove the`findUp` and `logger` imports from this file :+1:

**Comment by fredrikekelund** [comment]
Looks like I missed to run `nps validate` before pushing, I just ran `nps test`. Should be fixed now in any case!

**Comment by kentcdodds** [comment]
Would you like to [add yourself](../blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) to the [contributors table](../#contributors)?

**Comment by fredrikekelund** [comment]
No thanks, that's not necessary!

**Pull request merged**